backend/src/db/dataSourceWrapper.ts

```diff
@@ -140,6 +140,22 @@ export class DataSourceWrapper {
 		return releaseWithSprints[0]
 	}
 
+	public async getSprintWithBacklog(releaseId: number): Promise<Sprint[]> {
+		const sprints = await this.dataSource.getRepository(Sprint).find({
+			relations:{
+				release: true,
+				todos: true,
+			},
+			where: {
+				release: {id: releaseId}
+			},
```

Comment:
nice, i didnt know we could do that but this is a compact way of getting the sprints and backlog